Update backends.jl

nilsLue opened the pull request:
I have added arguments that are supported by a specific backend but not yet mentioned on the website.

nilsLue committed:
Update backends.jl

I have added arguments that are supported by a specific backend but not yet mentioned on the website.

nilsLue committed:
Update backends.jl

BeastyBlacksmith commented:
This PR is against your own fork. You want to make that PR against https://github.com/JuliaPlots/Plots.jl

BeastyBlacksmith commented on src/backends.jl:
```diff
@@ -740,7 +768,7 @@ const _pgfplots_attr = merge_with_base_supported([
     # :orientation,
     # :overwrite_figure,
-    :polar,
+    :polar;
```
thats probably unintentional

nilsLue committed:
Update backends.jl

Added more supported arguments by different backends. Copied pyplot attributes to pythonplot arguments so I can edit the attributes of pyplot because plot_title and similar attributes only work with the pyplot backend.
